### DISCUSSION TITLE
How can I run it other than port 80?

### DISCUSSION THREAD

**Discussion by hammyexe** [Hubro/clipface#55] [Q&A] [answered]
I tried this 
docker run -d \
  --name clipface \
  -v /home/hammy/clips:/clips \
  -p 3000:3000 \
  tomsan/clipface:latest

but I couldn't access it with port 3000

**Comment by Hubro** [comment, marked as answer]
Clipface listens on port 80 on the inside of the container, not port 3000. Try this instead:

```
docker run -d \
    --name clipface \
    -v /home/hammy/clips:/clips \
    -p 3000:80 \
    tomsan/clipface:latest
```

---

(You can change which port Clipface uses on the inside of the container by setting `-e PORT=12345`, but I'm not sure if there's any point in doing that)

**Reply by hammyexe** [reply]
Thank you it worked
